src/components/com_tjcertificate/site/views/certificate/tmpl/default.php

```diff
@@ -14,15 +14,22 @@
 use Joomla\CMS\Factory;
 use Joomla\CMS\Router\Route;
 use Joomla\CMS\Language\Text;
+use Joomla\CMS\Uri\Uri;
+use Joomla\CMS\HTML\HTMLHelper;
+
+HTMLHelper::StyleSheet('media/com_tjcertificate/vendors/font-awesome-4.1.0/css/font-awesome.min.css');
+HTMLHelper::script('media/com_tjcertificate/vendors/html2canvas/js/html2canvas.js');
+HTMLHelper::script('media/com_tjcertificate/js/tjCertificate.js');
 
```

Comment:
Load Media JS files like

```php
$options['relative'] = true;
HTMLHelper::script('com_tjcertificate/tjCertificate.js', $options);
```

Reply:
loaded for certificateImage.js but vendor js is not loading with adding "$options"